syntax = "proto3";

message Input {
    string headline = 1;
    string body = 2;
}

message Stance {
    string stance_class = 1;
    double pred_agree = 2;
    double pred_disagree = 3; 
    double pred_discuss = 4;
    double pred_unrelated = 5;
}

service FNCStanceDetection {
    rpc stance_detection(Input) returns (Stance) {}
}

